### ISSUE TITLE
Adding more features

### ISSUE THREAD

**Issue by gonzague** [bitfocus/companion-module-shure-p300#2]
Hello,

Does someone have the know-how to add more actions and feedbacks?

For example getting the current preset , getting the presets name,  getting the mute status, setting the mute status, get the USB connection status etc?

The [command strings list is given by Shure here](https://pubs.shure.com/command-strings/P300/en-US)

**Comment by josephdadams** [comment]
Yes, but unfortunately my free time is severely limited. I could add these for you as hired work. If you are interested, reach out to me via email. You can get my email address from my Github profile.
